integration/log_prep_test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Prepare a set of running processes for a Trillian log test.
+# This script should be loaded with ". integration/log_prep_test.sh",
```

Comment:
From what I can tell, our "sourcing" situation is this:

* ct_integration_test sources common and ct_prep_test
* ct_prep_test sources common, log_prep_test and ct_config
* log_prep_test sources common

(Just taking ct_integration_test as an example.)

Trap logic is also shared / spread between scripts.

It's a bit too much, IMO. I don't expect you to act on this now, but I'd like us to have a much simpler setup (1 script per test with one sourced at most). It may be for the best to pick another language to do this setup as well, if it proves to be too complex for bash (Python? Go?).

Better adherence to https://google.github.io/styleguide/shell.xml would be nice to have too.